src/test/java/com/mmm/his/cer/utility/farser/ast_complex/AstLimitationsTest.java

```diff
@@ -64,9 +64,22 @@ public void doesNotProperlyWork_OperatorWithParameters() throws Exception {
     assertThat(lines, is(new String[] {
         "IN",
         "  a",
-        "  1," // ... stuff is missing here
+        "  1" // ... stuff is missing here
     }));
 
   }
 
+  @Test
+  public void evaluateParenthesisWithinFunction() {
+    String input = ".containsStrings(a | (b & c), 2, 3)";
```

Comment:
ok, I can add a simpler one. Both your examples will fail because of the "(" within the function

Reply:
Hmm ok. I see, this would be the job of the Lexer to split these differently (know what is escaped, what is a string, ...) and this probably goes too far for the current use cases.